app/views/courses/_course.html.erb

```diff
@@ -8,10 +8,12 @@
     <%= course.status %>
   </p>
 
-  <%= link_to t(".view"), course, class: "btn btn-primary btn-sm" %>
+  <div style="float:right;">
```

Comment:
Avoid using inline style, you can write a class or id in css file and call it here